.travis/test.sh

```diff
@@ -2,20 +2,61 @@
 
 set -ev
 
+# Set up the build environment across Julia versions
+# This portion is the same as the default Travis test script
```

Comment:
Maybe include a link to the default Travis test script?

Reply:
https://github.com/travis-ci/travis-build/blob/master/lib/travis/build/script/julia.rb